### PR TITLE
Add impact of faster runners section to Discrouse post

### PR DESCRIPTION
**Pull request by welteki**
## Description

Add a new section with a table comparing build times for the tests workflow on a larger GithHub hosted runner vs the Hetzner AX102 Actuated runner that was used to run the tests for this article.

## Motivation and context

Show the overall impact of switching to faster runners and not only the speed improvements by switching the cache.

## How has this been tested.

Verified the post renders correctly.

### CODE REVIEW

File: posts/2024-02-23-local-caching-for-github-actions.md [hunk 1]
```diff
@@ -83,9 +83,31 @@ During our testing we noticed that every cache restore took a minimum of 10 seco
...
 
+## The impact of switching to faster runners
+
+The Discourse repo uses the larger GitHub hosted runners to run tests. The jobs we are going to compare are part of the [Tests workflow](https://github.com/discourse/discourse/actions/workflows/tests.yml). They are using runners with 8 CPUs and 32GB of ram so we replaced the `runs-on` label with an actuated label `actuated-8cpu-24gb` to run the jobs on similar sized microVMs.
+
+All jobs ran on the same [Hetzner AX102](https://www.hetzner.com/dedicated-rootserver/ax102/) bare metal host.
+
+This table compares the time it took to run each job on the hosted runner and on our Actuated runner. 
+
+| Job                  | GitHub hosted runner | Actuated runner |
```
Comment (alexellis) [inline posts/2024-02-23-local-caching-for-github-actions.md:94]: Third column with a %? or x speedup?

**Review by alexellis — COMMENTED** [review]
Thanks for adding this. Please see the note.

File: posts/2024-02-23-local-caching-for-github-actions.md (hunk 1 above)
Comment (welteki) [reply posts/2024-02-23-local-caching-for-github-actions.md]: A third column has been added.

**Review by alexellis — APPROVED** [review]
Approved

**Pull request merged**